node/src/proxy_server/mod.rs

```diff
@@ -4361,11 +4346,9 @@ mod tests {
             false,
         );
         let unaffected_socket_addr = SocketAddr::from_str("2.3.4.5:6789").unwrap();
-        let unaffected_stream_key =
-            StreamKey::new(main_cryptde().public_key().clone(), unaffected_socket_addr);
+        let unaffected_stream_key = StreamKey::default();
         let affected_socket_addr = SocketAddr::from_str("3.4.5.6:7890").unwrap();
-        let affected_stream_key =
-            StreamKey::new(main_cryptde().public_key().clone(), affected_socket_addr);
+        let affected_stream_key = StreamKey::default();
```

Comment:
Functionally, there's nothing wrong with this test; but it would be clearer to read if you used your `make_meaningful_stream_key()` function here with different parameters. If you don't know exactly what `StreamKey::default()` does, it looks like you're setting `unaffected_stream_key` and `affected_stream_key` to the same value.